### PR TITLE
[Merged by Bors] - chore: uniformize some author names

### PR DESCRIPTION
**Pull request by adomani**
I simply went for similarities: maybe some of these merges are actually different authors with very similar names!

Suggested by the list in [this Zulip message](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/copyright.20header.20line.20lint/near/454640469).

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/Set.lean
Comment (adomani) [inline Mathlib/Tactic/Set.lean:2]:
```suggestion
Copyright (c) 2022 Ian Benway. All rights reserved.
```

File: test/Set.lean
Comment (adomani) [inline test/Set.lean:2]:
```suggestion
Copyright (c) 2022 Ian Benway. All rights reserved.
```

File: Mathlib/Algebra/Order/Field/Pi.lean
Comment (YaelDillies) [inline Mathlib/Algebra/Order/Field/Pi.lean:4]: That's funny. Why am I the author but not the copyright holder?

**Review by YaelDillies — APPROVED** [review]

File: Mathlib/Algebra/Order/Field/Pi.lean
Comment (adomani) [reply Mathlib/Algebra/Order/Field/Pi.lean:4]: I'm not sure: maybe a weird previous split?

**Review by pechersky — APPROVED** [review]

**Review by dagurtomas — APPROVED** [review]

File: Mathlib/Algebra/Order/Field/Pi.lean
Comment (YaelDillies) [reply Mathlib/Algebra/Order/Field/Pi.lean:4]: 🤷

Comment (urkud) [reply Mathlib/Algebra/Order/Field/Pi.lean:4]: See leanprover-community/mathlib#17350

Comment (urkud) [reply Mathlib/Algebra/Order/Field/Pi.lean:4]: BTW, @YaelDillies am I right that the only lemma in this file doesn't need a (semi)field structure, only a linear ordered additive commutative group with dense order? If yes, could you please generalize it and move the file elsewhere?

Comment (YaelDillies) [reply Mathlib/Algebra/Order/Field/Pi.lean:4]: You are absolutely right: #15327. Not sure where to move the lemma though

**Comment by j-loreaux** [comment]
@adomani can you request a review from anyone whose name you changed that hasn't already approved the PR? I'm not saying we necessarily need approval from literally everyone, but the more the better.

**Comment by lecopivo** [comment]
I have lost my hats and dashes  "Tomáš Skřivan". Likely I'm not consistent about it but I would prefer the version with hats and dashes.

File: Mathlib/Tactic/TypeStar.lean
Comment (mattrobball) [inline Mathlib/Tactic/TypeStar.lean:4]:
```suggestion
Authors: Kyle Miller
```
I am not sure how I (alone!) ended up here but it was probably due to my general confusion.

**Review by mattrobball — APPROVED** [review]

**Comment by adomani** [comment]
> I have lost my hats and dashes "Tomáš Skřivan". Likely I'm not consistent about it but I would prefer the version with hats and dashes.

I simply made the replacement looking at the most frequent spelling that was used.  In your case, 9 were Tomas Skrivan and 5 were Tomáš Skřivan.

Anyway, I changed them all to Tomáš Skřivan!

**Review by alexjbest — APPROVED** [review]

**Review by fpvandoorn — APPROVED** [review]

**Review by mariainesdff — APPROVED** [review]

File: Mathlib/Topology/EMetricSpace/Paracompact.lean
Comment (vihdzp) [inline Mathlib/Topology/EMetricSpace/Paracompact.lean:2]: I think the name might not be the only issue here...

File: Mathlib/Data/Finsupp/AList.lean
Comment (vihdzp) [inline Mathlib/Data/Finsupp/AList.lean:2]: This is indeed my file and that is indeed my full name, thanks!

File: Mathlib/Topology/EMetricSpace/Paracompact.lean
Comment (adomani) [reply Mathlib/Topology/EMetricSpace/Paracompact.lean:2]: Yury is a pioneer of formalization.

**Comment by adomani** [comment]
This PR has been open for about 2 weeks.

I think that everyone who wanted to object to how their name is spelled probably already commented, so I removed the `awaiting-author`s label!

File: Mathlib/Topology/EMetricSpace/Paracompact.lean
Comment (Ruben-VandeVelde) [inline Mathlib/Topology/EMetricSpace/Paracompact.lean:2]:
```suggestion
Copyright (c) 2021 Yury Kudryashov. All rights reserved.
```
https://github.com/leanprover-community/mathlib3/pull/6395

**Review by Ruben-VandeVelde — APPROVED** [review]
lgtm. Please maintainer merge on my behalf

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**
